client/my-sites/checkout/checkout/index.jsx

```diff
@@ -372,6 +371,17 @@ export class Checkout extends React.Component {
 		return true;
 	}
 
+	/**
+	 * Purchases are of the format { [siteId]: [ { productId: ... } ] }
+	 * so we need to flatten them to get a list of purchases
+	 *
+	 * @param {object} purchases keyed by siteId { [siteId]: [ { productId: ... } ] }
+	 * @returns {Array} of product objects [ { productId: ... }, ... ]
+	 */
+	flattenPurchases( purchases ) {
```

Comment:
Probably a merge issue, but I think this change shouldn't be there.

Reply:
Yes, looks like it from here https://github.com/Automattic/wp-calypso/commit/49cdb5ca2f05930d3dbc98a8ea77dbef52cbed1e. Removed in https://github.com/Automattic/wp-calypso/pull/43199/commits/6b7f8bf727b28a3292892752ef807df4f2b9926a